javascript/javascript2/week1/animal-spirit/main.js

```diff
@@ -0,0 +1,45 @@
+// functions 
+function getAnimalNumber() {
+  let max = spiritualtraits.length;
+  return Math.floor(Math.random() * (max  + 1)) ;
+}
+
+function getAnilmalTraitsIndex() {
+  let max = 3;
+  return Math.floor(Math.random() * (max + 1)) ;
+}
+
+function getYourSpiritualAnimal() {
+  let animalIndex = getAnimalNumber();
+  let TraitsIndex = getAnilmalTraitsIndex();
+  let spices = spiritualtraits[animalIndex].traits[TraitsIndex];
+  let animalName = spiritualtraits[animalIndex].name;
```

Comment:
I see this as very different approach, I appreciate your efforts. This exercise can be simplified.

All the variables declared here can be used as const, as I do not see that you are actually reassigning anything to them.

Reply:
yes, I know that the exercise can be simplified but I thought it will be good training and a bit more challenging.